cmd/auto

```diff
@@ -41,7 +41,7 @@ egrep $regex $CFILE | while read iopt; do
     key=${BASH_REMATCH[2]}
     val=${BASH_REMATCH[3]}
     if [ "$key" = opts ]; then
-        start_faux $iface $(eval echo $val)
+        start_faux $iface $val
```

Comment:
I'm not sure what the eval was intended for but it's not passing the the "-n" option correctly.

Reply:
Seems like the kind of thing that would benefit from a comment as to why it's there! (Likely my fault).

For now just do whatever you need for the tests to pass and/or it to work, and then if we run into the situation where it's required again. And then make sure to add the comment.